lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -1677,33 +1671,33 @@ namespace Js
         if (!scriptContext->GetConfig()->IsES6RegExSymbolsEnabled())
         {
             JavascriptRegExp * regExObj = JavascriptRegExp::CreateRegEx(regExp, nullptr, scriptContext);
-            return RegexHelper::RegexSearch(scriptContext, regExObj, pThis);
+            return fallback(regExObj, pThis);
         }
         else
         {
             JavascriptRegExp * regExObj = JavascriptRegExp::CreateRegExNoCoerce(regExp, nullptr, scriptContext);
-            Var searcher = GetRegExSymbolSearch(regExObj, scriptContext);
-            return CallRegExSymbolSearch(searcher, regExObj, args[0], varName, scriptContext);
+            Var symbolFn = GetRegExSymbolFunction(regExObj, symbolPropertyId, scriptContext);
+            return CallRegExSymbolFunction(symbolFn, regExObj, args[0], varName, scriptContext);
         }
     }
 
-    Var JavascriptString::GetRegExSymbolSearch(Var regExp, ScriptContext* scriptContext)
+    Var JavascriptString::GetRegExSymbolFunction(Var regExp, PropertyId propertyId, ScriptContext* scriptContext)
     {
         return JavascriptOperators::GetProperty(
             RecyclableObject::FromVar(JavascriptOperators::ToObject(regExp, scriptContext)),
-            PropertyIds::_symbolSearch,
+            propertyId,
             scriptContext);
     }
 
-    Var JavascriptString::CallRegExSymbolSearch(Var search, Var regExp, Var string, PCWSTR const varName, ScriptContext* scriptContext)
+    Var JavascriptString::CallRegExSymbolFunction(Var fn, Var regExp, Var string, PCWSTR const varName, ScriptContext* scriptContext)
     {
-        if (!JavascriptConversion::IsCallable(search))
+        if (!JavascriptConversion::IsCallable(fn))
         {
             JavascriptError::ThrowTypeError(scriptContext, JSERR_FunctionArgument_Invalid, varName);
         }
 
-        RecyclableObject* searchFn = RecyclableObject::FromVar(search);
-        return searchFn->GetEntryPoint()(searchFn, CallInfo(CallFlags_Value, 2), regExp, string);
+        RecyclableObject* fnObj = RecyclableObject::FromVar(fn);
+        return fnObj->GetEntryPoint()(fnObj, CallInfo(CallFlags_Value, 2), regExp, string);
```

Comment:
Looks like I had to tag the function in order for it to show up in the call stack (commit fd4c49827d50b351d72fb13c85614dd0bd15fc4c). Thanks @akroshg for pointing this out.